protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -262,7 +262,7 @@ public function actionAdminUserSearch($keyword, $id)
     {
         $group = Group::findOne($id);
```

Comment:
Please change to `findOne(['id' => $id])`

Reply:
Done!